__tests__/database/schema/UserSchema.test.ts

```diff
@@ -512,7 +514,19 @@ describe('TESTING UserSchema', () => {
           profile_pic_path: 'profilePicture',
           updated_at: 'updatedAt',
           username: 'username',
-          is_banned: false
+          is_banned: false,
+          last_active_at: 'lastActiveAt',
+          user: {
+            banned: false,
+            id: 'userId',
+            name: 'username',
+            username: 'username',
+            last_active_at: 'lastActiveAt',
+            image: 'profilePicture',
+            image:
+              'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+            created_at: 'createdAt'
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a duplication of fields inside the 'user' object (e.g., 'username', 'image'). Also, the 'image' field is defined twice, which could lead to unexpected behavior. Ensure that each key in the object is unique and necessary.
```diff
          profile_pic_path: 'profilePicture',
          updated_at: 'updatedAt',
          username: 'username',
          is_banned: false,
          last_active_at: 'lastActiveAt',
          user: {
-            banned: false,
-            id: 'userId',
-            name: 'username',
-            username: 'username',
-            last_active_at: 'lastActiveAt',
-            image: 'profilePicture',
-            image:
-              'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
-            created_at: 'createdAt'
+            banned: false,
+            id: 'userId',
+            name: 'username',
+            last_active_at: 'lastActiveAt',
+            image: process.env.DEFAULT_PROFILE_PICTURE_URL,
+            created_at: 'createdAt'
          }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->